app/views/accounts/_objective.html.slim

```diff
@@ -2,6 +2,11 @@ tr
   td= objective.name
   td= objective.amount
   td= number_to_percentage(account.balance / objective.amount * 100, precision: 0)
-  td= ([objective.amount - account.balance, 0].max / account.automatic_topup_configs.sum(:amount).+(0.1)).round(2)
-  td= image_tag objective.image_url
+  - if objective.amount > account.balance
+    - if (( objective.amount - account.balance )/objective.amount) * 100 > 10
+      td= weeks_to_goal(objective.amount, account)
+    - else
+      td= 'soon'
+  - else
+    td= ''
```

Comment:
What does it mean ""?

If balance is say 10, but goal is 1000, and weekly amount is 100 then it should "". Why?

Reply:
Ah got it, seems that's not the case. And it makes sense. It's still not clear why I see nothing in the example above.

Reply:
- When we reach the goal amount, in other word when `objective.amount < account.balance` then the "weeks to the goal" loses its meaning. (here we have 2 x if and 2 x else!)

Reply:
It's not the only issue. Let's discuss